Unable to resolve path to module 'underscore.string/titleize'  import/no-unresolved

In [CI build 1529](https://circleci.com/gh/mozilla/normandy/1529?utm_campaign=build-failed&utm_medium=email&utm_source=notification) on master, this error came up:

```
/app/code/recipe-server/client/control/actions/FilterActions.js
  12:22  error  Unable to resolve path to module 'underscore.string/titleize'  import/no-unresolved
```

The build in the PR also had the error, but the error was not reported because of #481.

I don't think we have the underscore library installed that can support this sort of piece-meal usage. I also didn't find `titleize` in the version of underscore installed in recipe-server.

--- a/lints/package.json
+++ b/lints/package.json
@@ -33,7 +33,8 @@
     "redux-thunk": "2.1.0",
     "sha.js": "2.4.5",
     "uglifyjs": "2.4.10",
-    "underscore": "1.8.3"
+    "underscore": "1.8.3",
+    "underscore.string": "3.3.4"
   },
   "devDependencies": {
     "babel-core": "6.14.0",

--- a/recipe-server/package.json
+++ b/recipe-server/package.json
@@ -44,7 +44,7 @@
     "sha.js": "2.4.5",
     "uglifyjs": "2.4.10",
     "underscore": "1.8.3",
-    "underscore.string": "^3.3.4"
+    "underscore.string": "3.3.4"
   },
   "devDependencies": {
     "babel-core": "6.14.0",

